### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by KieranHons**
Feel free to give input :) Docs are tough mkay

### CODE REVIEW

File: docs/quantum-mechanic/command-reference.md [hunk 1]
```diff
@@ -54,7 +60,16 @@ Links a library contained within a BitBucket project to the project.
 Links an application contained within a BitBucket project to the project.
 
 ### **associate team**
-//TODO
+Links a team to an existing project. 
 
 ### **jenkins build**
-Starts a jenkins build with the current project.
\ No newline at end of file
+Starts a jenkins build with the current project.
```
Comment (kbristow) [inline docs/quantum-mechanic/command-reference.md]: with the selected project

File: docs/tenants.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# **Tenants**
+Tenants allow projects spanning multiple bitbucket projects, teams and actual business groupings to be connected.
```
Comment (kbristow) [inline docs/tenants.md]: A whole page for this? :P

File: docs/user-guide/add-an-application.md [hunk 3]
```diff
@@ -9,14 +9,23 @@ Below is an example of linking the existing application to your project
 
 ![Add Application](/images/user-guide/add-an-application/add-application.png)
 
-The flow for adding an existing application is different to all the commands encounterd so far. First the prompts for an application name and description are shown. Enter these and submit them when you are satisfied with your values. Subatomic will then pull a list of all the repositories in your Bitbucket project and you can select which one is holds the source for the application you wish to use. Once you have made you selection you will be informed that your application is being provisioned.
+The flow for adding an existing application is different to all the commands encounterd so far. First the prompts for an application name and description are shown. Enter these and submit them when you are satisfied with your values. Subatomic will then pull a list of all the repositories in your Bitbucket project and you can select which one holds the source for the application you wish to use. Once you have made your selection you will be informed that your application is being provisioned.
```
Comment (kbristow) [inline docs/user-guide/add-an-application.md]: I think say "application is being created" here. Provisioned in for us usually refers to the environment creation etc...

File: docs/tenants.md (hunk 2 above)
Comment (kbristow) [reply docs/tenants.md]: Maybe use a little blurb below the command/user guide descriptions. The same way we add side information for see this command reference here thing with this blurb?

Comment (KieranHons) [reply docs/tenants.md]: Don't want the tenants to be upset :P

**Pull request merged**